rfc021-platform-support-policy.md

```diff
@@ -48,14 +48,11 @@ Ubuntu Linux | 12.04, 14.04 | x86, x86_64 | deb | Ubuntu 12.04
 
 Tier 2 supported platforms are those on which Omnitruck will serve packages, but those packages may not have been built on that OS variant. Additionally, we may or may not do post-build verification on these platforms. A platform must have at least one Maintainer before being added to this tier.
 
-* SUSE Linux Enterprise Server 10, 11
```

Comment:
Please update Appendix 2: Timeline of Platform Addition/Removal with the removal of 10.